README.md

```diff
@@ -1,8 +1,13 @@
 # Ruby Practice
 ## String
 - String multiplication
+    - For this assignment we would like you to use you to use your best Santa impression and multiply "Ho" three times to have an output of "HoHoHo"
+
 - upcase, downcase, swapcase
+    - Using the "greeting, peole and other_people" variables display "HELLO friends AnD FaMiLy" using a combination of the `upcase`, `downcase`, and `swapcase` methods. 
+
 - chomp
+    - Using the `chomp` method take the "greeting" variable and return an output of "Hello!"
```

Comment:
I think having it in the REAME is also good. They can see the details of each assignment so they can pick what order they want to tackle them in.

Reply:
@jelaniwoods I agree I think that having it in both places will be good.  I think I just need to make a note in the readme to not uncomment the comments.